panics_test.go

```diff
@@ -917,6 +917,18 @@ func TestSetOutputNilWriter(t *testing.T) {
 	fmt.Fprintln(output.Load(), "DO NOT PANIC")
 }
 
+func TestWriteTimeout(t *testing.T) {
+	testSetup(t)
+
+	if writeTimeout.Load() != int64(500*time.Millisecond) {
+		t.Errorf("Expected default writeTimeout to be set to 100ms, but got %d.", time.Duration(writeTimeout.Load()))
+	}
+	WriteTimeout(90 * time.Millisecond)
+	if writeTimeout.Load() != int64(90*time.Millisecond) {
+		t.Errorf("Expected writeTimeout to be set to 90ms, but got %d.", time.Duration(writeTimeout.Load()))
+	}
+}
```

Comment:
We still need to test the behavior of WriteTimeout (this only tests that an `atomic.Int64` works as expected). Ideally, we test that when WriteTimeout is `<= 0` that `Capture` does not return until the full stack trace was written (the `WriterFunc` adapter will be helpful here). More specifically, we should test that the writer returns before the `Capture` and I think we can and should achieve this without having to use long sleeps (anything timing dependent makes for slow tests that can be flaky in CI).